src/js/components/atoms/c-image/c-image.vue.html

```diff
@@ -1,4 +1 @@
-<div>
-    <img :alt="alt" :src="src" :srcset="srcset" class="img--contain" :class="classList" @error="setError"/>
-    <div class="img--contain img--replacement" v-if="imageError">{{ alt }}</div>
-</div>
+<img :alt="alt" :src="src" :srcset="srcset" class="img--contain" :class="classList" @error="setError"/>
```

Comment:
The `@error="setError"` is left in on purpose?

Reply:
yes, the method still sets the error in the component, which can be used later to try and sort out the error handling in the component itself, but also emits the error so the parent component can also deal with it if necessary - see card-product